modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/dataframe.py

```diff
@@ -1120,6 +1120,10 @@ def add_col_field(table, col_name, table_col_name):
                     and (not pyarrow.types.is_null(field.type))
                     and (
                         pyarrow.types.is_string(field.type)
+                        or (
```

Comment:
Why check `bit_width` field?

Reply:
If the bit_width is equal, but one of the fields is of a floating point type, we choose the floating.

Reply:
It would be nice to add documentation to `add_col_field` function to understand what it does and why.

Reply:
Done.